salt/search/aws.sls

```diff
@@ -0,0 +1,10 @@
+aws-credentials-cli:
```

Comment:
the name of this state is misleading as these credentials are used primarily by the PHP SDK (and apparently also by the aws cli tool)